usaon_vta_survey/templates/response/relationships/application_societal_benefit_area.html

```diff
@@ -49,31 +48,8 @@ <h4>Societal Benefit Areas</h4>
 
   <form method="POST">
     {{form.csrf_token}}
+    {{render_form(form)}}
 
-    <div class="two-columns">
-      <div>
-        {% if sqla_inspect(application).transient %}
-          {{ application_fields(form.application.form) }}
-        {% endif %}
-      </div>
-
-      <div>
-        {% if sqla_inspect(societal_benefit_area).transient %}
-          {{ societal_benefit_area_fields(form.societal_benefit_area.form) }}
-        {% else %}
-          <p>Name: {{societal_benefit_area.societal_benefit_area_id}}</p>
-        {% endif %}
-      </div>
-    </div>
-
-    <h4>Relationship</h4>
-    {% if sqla_inspect(relationship).transient %}
-      {{ application_societal_benefit_area_fields(form.relationship.form) }}
-    {% else %}
-      {{relationship}}
-    {% endif %}
-
-    <input type="submit" value="Create new relationship" />
```

Comment:
Love seeing all these deletions :)

Reply:
@MattF-NSIDC it's still showing changes requested, but I think they're all covered. Let me know if there's anything else?

Reply:
If you click the refresh icon next to the "changes requested" red icon, you can clear it.